### PR TITLE
Add final code together at end of page

### PR DESCRIPTION
**Pull request by pbouffard**
For the impatient.

### CODE REVIEW

**Review by SimonDanisch — DISMISSED** [review]

File: docs/src/makielayout/tutorial.md [hunk 1]
```diff
@@ -532,3 +532,99 @@ documentation for more details and other dynamic parts like sliders and buttons!
 using GLMakie
 GLMakie.activate!()
 ```
+
+## Final code
+Here is the code producing the final figure above:
```
Comment (jkrumbiegel) [inline docs/src/makielayout/tutorial.md:537]: Maybe add:

"Note that this code is not perfect, but simply reflects the tutorial content including unnecessary intermediate steps like misplacing and then moving the legend."

**Review by jkrumbiegel — CHANGES_REQUESTED** [review]

File: docs/src/makielayout/tutorial.md (hunk 1 above)
Comment (pbouffard) [reply docs/src/makielayout/tutorial.md:537]: Not sure what you mean by the unnecessary intermediate steps, can you elaborate?

Comment (jkrumbiegel) [reply docs/src/makielayout/tutorial.md:537]: Well I deliberately place the legend in the "wrong" place first to show that you can move it, but of course if you run the whole script at once it's a bit silly to include that. But it's not supposed to be streamlined code, just the collection of all steps

Comment (jkrumbiegel) [reply docs/src/makielayout/tutorial.md:537]: now you removed that step, but I wouldn't want the code down there to be different. just tell people that it's not how the "ideal" code would be written. it's the same with the heatmap example, they are also moved after creation.

Comment (jkrumbiegel) [reply docs/src/makielayout/tutorial.md:537]: it's a kind of tutorial where some less than ideal things are shown first and then resolved

**Comment by SimonDanisch** [comment]
@jkrumbiegel I guess we only need a short notice before the code like:
```
Note that this code is just copy & pasted from the tutorial steps, and not representative of how you'd write it in one go.
```
To merge this?

**Comment by jkrumbiegel** [comment]
ok I had thought about it a bit more, and I've added something similar with the updated code to the beginning of the page, but in a detail view, so click to expand. I think that's visible but also doesn't get in the way. https://github.com/JuliaPlots/AbstractPlotting.jl/pull/745 I'll close this PR then, thanks for sticking with it

**Pull request closed without merging**